### PR TITLE
Feat/add ricardian content

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do ?
Add clauses and contracts explanation of a Ricardian Contract

### Steps to test
1. yarn install
1. yarn start
1. go to http://localhost:3000/docs/eos-learn/ricardian-contracts/

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by AngeloCG97 — COMMENTED** [review]
Review conflicts

File: docs/eos-learn/ricardian-contracts.md [hunk 1]
```diff
@@ -4,20 +4,59 @@ title: Ricardian contracts
...
 - Human and software readable.
 - It can be printed.
 - All forms of documents (physical, virtual, encrypted) are equivalent.
 - They are signed by all parties.
 - They can be identified safely thanks to the has of each contract.
 
-### **Ricardian contracts at EOSIO**
+### Ricardian contracts at EOSIO
 
 Blockchains operate under smart contracts, however, work is underway to include Ricardian contracts in the transactions of these networks. One of the main objectives is to convert the complexity of the code observed within Blockchain technology into a simpler language so that anyone could understand it.
```
Comment (xavier506) [inline docs/eos-learn/ricardian-contracts.md:20]: lets include a link to view an example Ricardian contract on our components storybook:

https://components.eosio.cr/?path=/story/ricardiancontract--ricardian-contract

**Review by xavier506 — CHANGES_REQUESTED** [review]
Eos costa rica has developed a react component that renders ricardian contracts for smart contracts on EOSIO networks. You can view the react component in the foollowing link.

https://components.eosio.cr/?path=/story/ricardiancontract--ricardian-contract

the open source repository for  this component is located here:  https://github.com/eoscostarica/eoscr-components

File: docs/eos-learn/ricardian-contracts.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ They are digital documents that define the terms and conditions among those invo
 ### Ricardian contracts at EOSIO
 Blockchains operate under smart contracts, however, work is underway to include Ricardian contracts in the transactions of these networks. One of the main objectives is to convert the complexity of the code observed within Blockchain technology into a simpler language so that anyone could understand it.
 
-Eos costa rica has developed a react component that renders ricardian contracts for smart contracts on EOSIO networks. You can view the react component in the following [link](https://components.eosio.cr/?path=/story/ricardiancontract--ricardian-contract). In that link, go to the `Knobs` section to modify `httpEndpoint` and `contractName`. Once done, check the Ricardian Contract for a Smart Contract. For example, set:
+EOS COSTA RICA has developed a react component that renders ricardian contracts for smart contracts on EOSIO networks. You can view the react component in the following [link](https://components.eosio.cr/?path=/story/ricardiancontract--ricardian-contract). In that link, go to the `Knobs` section to modify `httpEndpoint` and `contractName`. Once done, check the Ricardian Contract for a Smart Contract. For example, set:
```
Comment (xavier506) [inline docs/eos-learn/ricardian-contracts.md:21]: EOS Costa Rica

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**